components/meta-packages/sunray-essential/sunray-essential.p5m

```diff
@@ -0,0 +1,107 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025, Carsten Grzemba. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+# graphical desktop
+depend fmri=mate_install type=require
+
+# DHCP service
+depend fmri=service/network/dhcp/isc-dhcp type=require
+# empty package only for satisfy installation requirements
+# SRSS on Solaris uses the old Sun DHCP server but on OI we use ISC-DHCP
+depend fmri=service/network/dhcp type=require
+
+# SUNWutkio-libs package dependency, empty packages only
+# for satisfy installation requirements
+depend fmri=consolidation/X/X-incorporation type=require
+depend fmri=compatibility/packages/SUNWxwplt type=require
+
+# for Sun JRE
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libCrun.so.1
+
+# SUNWut configure requirements rpm2cpio, see docs.openindiana.org
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/bin/rpm2cpio
+
+# in OI legacy/retired 64bit packages
+# GDM and Gnome greeter, only gdm-session-worker has to run in 32bit
+#
+# pkg.debug.depend.file does not working yet, there is no appropriate package in the OI repo
+# depend type=require fmri=__TBD pkg.debug.depend.file=usr/sbin/gdm-binary
+depend type=require fmri=system/display-manager/gdm@2.30.7
```

Comment:
Isn't this essentially the same as the dependency on `usr/sbin/gdm-binary` above?

There is no `system/display-manager/gdm` package (it was obsoleted) so how this line is supposed to work?  Similarly for few other packages below.

Reply:
Sure, I mentioned this in my last PR comment.